inference_schema/parameter_types/numpy_parameter_type.py

```diff
@@ -36,6 +37,9 @@ def __init__(self, sample_input, enforce_column_type=True, enforce_shape=True):
         self.enforce_column_type = enforce_column_type
         self.enforce_shape = enforce_shape
 
+    def supported_versions(self):
```

Comment:
Since all of the parameter types are the same you can move this to the abstract class, for common code reuse